src/sdk/namespace_fs.js

```diff
@@ -1118,7 +1118,9 @@ class NamespaceFS {
         const fs_context = this.prepare_fs_context(object_sdk);
         const open_mode = 'w';
         try {
-            let MD5Async = config.NSFS_CALCULATE_MD5 ? new (nb_native().crypto.MD5Async)() : undefined;
+            const namespace_bucket_config = await object_sdk.read_bucket_sdk_namespace_info(params.bucket.name)
+            let MD5enabled = namespace_bucket_config.write_resource.nsfs_md5_enabled 
+            let MD5Async = MD5enabled ? new (nb_native().crypto.MD5Async)() : undefined;
```

Comment:
style: use `const` instead of `let` where possible, and use snake_case for identifier names.

Reply:
ack , Thanks